e2e/app/page/profile-page.ts

```diff
@@ -1,25 +1,32 @@
-import {Page} from 'puppeteer';
+import {Page, WaitForSelectorOptions} from 'puppeteer';
 import Textbox from 'app/element/textbox';
 import AuthenticatedPage from 'app/page/authenticated-page';
 import {waitWhileLoading} from 'utils/test-utils';
 import {waitForDocumentTitle, waitForUrl} from 'utils/waits-utils';
+import Button from '../element/button';
+import Textarea from '../element/textarea';
+import BaseElement from '../element/base-element';
 
 export const PageTitle = 'Profile';
 
 export const LabelAlias = {
-  FirstName: 'First Name',
-  LastName: 'Last Name',
-  ContactEmail: 'Contact Email',
-  CurrentPosition: 'Your Current Position',
-  Organization: 'Your Organization',
-  CurrentResearchWork: 'Current Research Work',
-  AboutYou: 'About You',
-  Institution: 'Institution',
-  Role: 'Role',
-  DiscardChanges: 'Discard Changes',
+  ResearchBackground: 'Your research background, experience and research interests',
   SaveProfile: 'Save Profile',
 };
 
+
+export const DataTestIdAlias = {
```

Comment:
`export` is not needed if `DataTestIdAlias` isn't going to be imported by any outside file.